components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/latest/GitpodCopyWebUrlAction.kt

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package io.gitpod.jetbrains.remote.latest
+
+import com.intellij.ide.BrowserUtil
+import com.intellij.notification.NotificationAction
+import com.intellij.notification.NotificationType
+import com.intellij.openapi.actionSystem.ActionPlaces
+import com.intellij.openapi.actionSystem.ActionUpdateThread
+import com.intellij.openapi.actionSystem.AnAction
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.components.service
+import com.intellij.openapi.ide.CopyPasteManager
+import com.intellij.util.application
+import com.jetbrains.rd.platform.codeWithMe.portForwarding.PortForwardingDataKeys
+import io.gitpod.jetbrains.remote.GitpodManager
+import io.gitpod.supervisor.api.Status.PortsStatusRequest
+import io.gitpod.supervisor.api.StatusServiceGrpc
+import kotlinx.coroutines.launch
+import java.awt.datatransfer.StringSelection
+
+@Suppress("ComponentNotRegistered", "UnstableApiUsage")
+class GitpodCopyWebUrlAction : AnAction() {
+    override fun actionPerformed(e: AnActionEvent) {
+        e.dataContext.getData(PortForwardingDataKeys.SUGGESTION)?.getSuggestedHostPort()?.let { hostPort ->
+            application.coroutineScope.launch {
+                getUrlFromPort(hostPort)?.let {
+                    CopyPasteManager.getInstance().setContents(StringSelection(it))
+                    service<GitpodManager>()
```

Comment:
In other actions where we use clipboard (share snapshot), we don't send a notification to tell the user that the action is successfull, I guess it's explicit in most UIs that if you click "Copy" it will work and copy to your clipboard?

VS Code ports tab offers a copy action too and we don't notify.

Maybe it would be less intruisive if we don't send any notification?

Thoughts? @felladrin @akosyakov